packages/cpt-ui/src/pages/CookiePolicyPage.tsx

```diff
@@ -106,24 +102,16 @@ const CookiePolicyPage = () => {
     <div className="nhsuk-width-container nhsuk-u-margin-top-4">
       <main className="nhsuk-main-wrapper nhsuk-main-wrapper--s" id="main-content" role="main">
         <nav className="nhsuk-breadcrumb" aria-label="Breadcrumb">
```

Comment:
This should use the proper react component from the library: https://main--65aa76b29d00a047fe683b95.chromatic.com/?path=/docs/navigation-breadcrumb--docs